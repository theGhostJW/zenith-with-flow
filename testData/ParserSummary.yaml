rawFile: 'C:\ZWTF\testData\DemoLog.raw.yaml'
elementsFile: 'C:\ZWTF\logs\DemoLog.elements.yaml'
testSummaries:
  Another_Demo_Case:
    testConfig:
      script: Another_Demo_Case
      id: 2
      when: i test anther case
      then: it still works
      owner: JW
      enabled: true
      countries:
        - Australia
      environments:
        - TST
      depth: Regression
    startTime: '2017-11-25 11:28:22'
    endTime: '2017-11-25 11:28:22'
    stats:
      iterations: 2
      passedIterations: 1
      failedIterations: 1
      iterationsWithWarnings: 1
      iterationsWithKnownDefects: 0
  Demo_Case:
    testConfig:
      script: Demo_Case
      id: 1
      when: I cannot think of anything
      then: I just write something
      owner: JW
      enabled: true
      countries:
        - New Zealand
        - Australia
      environments:
        - TST
      depth: Regression
    startTime: '2017-11-25 11:28:22'
    endTime: '2017-11-25 11:28:22'
    stats:
      iterations: 4
      passedIterations: 1
      failedIterations: 3
      iterationsWithWarnings: 0
      iterationsWithKnownDefects: 1
  Demo_Case3:
    testConfig:
      script: Demo_Case3
      id: 2
      when: i test anther case
      then: it still works
      owner: JW
      enabled: true
      countries:
        - Australia
      environments:
        - TST
      depth: Regression
    startTime: '2017-11-25 11:28:22'
    endTime: '2017-11-25 11:28:22'
    stats:
      iterations: 1
      passedIterations: 1
      failedIterations: 0
      iterationsWithWarnings: 0
      iterationsWithKnownDefects: 1
runSummary:
  runConfig:
    name: Test Test Run
    mocked: false
    country: Australia
    environment: TST
    testCases: []
    depth: Regression
  startTime: '2017-11-25 11:28:22'
  endTime: '2017-11-25 12:38:44'
  filterLog:
    Another_Demo_Case: Accepted
    Demo_Case: Accepted
  stats:
    testCases: 3
    passedTests: 1
    failedTests: 2
    testsWithWarnings: 1
    testsWithKnownDefects: 2
    testsWithType2Errors: 1
    iterations: 7
    passedIterations: 3
    failedIterations: 4
    iterationsWithWarnings: 1
    iterationsWithType2Errors: 1
    iterationsWithKnownDefects: 2
    outOfTestErrors: 1
    outOfTestWarnings: 2
    outOfTestType2Errors: 1
    outOfTestKnownDefects: 1
