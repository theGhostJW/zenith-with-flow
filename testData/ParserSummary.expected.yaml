################################################################################
########################### Summary - Test Test Run ############################
################################################################################

start:    2017-11-25 11:28:22
end:      2017-11-25 12:38:44
duration: 01:10:22
raw:      .\DemoLog.raw.yaml

runConfig:
  mocked:      false
  country:     Australia
  environment: TST
  testCases:   []
  depth:       Regression

stats:
  testCases:                  30
  passedTests:                 1
  failedTests:                 2
  testsWithWarnings:           1
  testsWithKnownDefects:       2
  testsWithType2Errors:        1

  iterations:                  7
  passedIterations:            3
  failedIterations:            4
  iterationsWithWarnings:      1
  iterationsWithType2Errors:   1
  iterationsWithKnownDefects:  2

  outOfTestErrors:             1
  outOfTestWarnings:           2
  outOfTestType2Errors:        1
  outOfTestKnownDefects:       1
